### ISSUE TITLE
Drafting of New Elicitation Mechanisms

### ISSUE THREAD

**Issue by xehu** [Watts-Lab/task-mapping#314]
- [ ] @xehu - Set up a place where we can draft new questions and discuss with each other
- [ ] Draft cleaned elicitation mechanisms for each of the taxonomies:
> - [ ] McGrath
> - [ ] Laughlin
> - [ ] Shaw
> - [ ] Steiner
> - [ ] Wood
> - [ ] Zigurs

- [ ] Move cleaned elicitation mechanism to Qualtrics

**Comment by markwhiting** [comment]
@xehu this is somewhat useful → https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories

Looks like we can create custom categories or labels (which I already did just as an example) on a per taxonomy level.

**Comment by xehu** [comment]
@markwhiting Thank you! I just created threads + a custom label for everything. It's probably overkill to create a custom category for each one as well, so I just put them all under 'General.'

**Comment by xehu** [comment]
@linneagandhi Can I ask for your help on something? In my recent work, I realize that I didn't read either Wood or Zigurs --- I had focused more on the other ones. I'm not even 100% sure if I'm pulling the correct papers. This is what I found for them:

- Wood: https://www.researchgate.net/profile/Robert-Wood-40/publication/222383252_Task_complexity_Definition_of_the_construct/links/5e8ec06b4585150839cad28d/Task-complexity-Definition-of-the-construct.pdf
- Zigurs: https://www.jstor.org/stable/pdf/249668.pdf?casa_token=TvJBZkEr1wMAAAAA:pN3Q8VupA3f28lqInp2b3z4zIPJ2PducFtqUovFElr-chRw0d2ebHW_ENBy13EHm70KnuzHuiYIYv9ytcRpkb0saMfhJYOM_8Hq0nAkm3XWz8-4ulkw

These are the papers I'm currently linking on the [Wood discussion post](https://github.com/Watts-Lab/task-mapping/discussions/323) and [Zigurs discussion post](https://github.com/Watts-Lab/task-mapping/discussions/325), respectively.

Could you please confirm that these are the papers from which you wrote the original questions?

**Comment by linneagandhi** [comment]
@xehu sorry I don't have any automated way to see these notifications so just saw this!

I used Wood (1986) - this is on Abdullah's mindmanager chart and Zigurs (1999) from the original questions list Abdullah made (see "questions" tab). The source for Zigurs I believe is [here](https://dl.acm.org/doi/pdf/10.1145/344241.344244?casa_token=xn1jyYoCd_gAAAAA:KlGuqC0jvgFaQWgKI3wZv3LkGxjl98F5skNN8x2y_zlj89WOPYfL1bI9T_dqDGWGolb1WiKTOU4)

**Comment by linneagandhi** [comment]
And Wood 1986 is [here] which I got from the project folder (https://www.sciencedirect.com/science/article/pii/0749597886900440#:~:text=Task%20complexity%3A%20Definition%20of%20the%20construct%E2%98%86&text=A%20general%20theoretical%20model%20of,)%20acts%2C%20and%20information%20cues.&text=Various%20indexes%20which%20can%20be,these%20constructs%20are%20also%20presented.)

**Comment by linneagandhi** [comment]
The starting point for all questions however was the original map by Abdullah which it seems cannot upload here - I'll send it by slack

**Comment by amaatouq** [comment]
Yes, the Woods paper is "[Task complexity: Definition of the construct](https://www.sciencedirect.com/science/article/abs/pii/0749597886900440)."

The only taxonomies that I know of are the ones on the mindmanager. Zigurs is the only one that's not on the spaghetti diagram and so I don't know where it came from or when/how (or by whom) it was added to the [Questions sheet](https://docs.google.com/spreadsheets/d/1btASh4JtkWEWxO3G7yaehysZ-lyysylQ4fTe3yKn64M/edit?usp=sharing).

**Comment by xehu** [comment]
Drafts of the new questions + pulled quotes from every paper are up in the [spreadsheet](https://docs.google.com/spreadsheets/d/1cKqeOE7kf3bY3shtn9lep_W2ya1uao8hwx_qK2_jQs0/edit#gid=1735245561) now --- so the next step is to dogfood?

Consolidating some questions (some of this is discussed briefly in Slack) we may want to answer in terms of next steps:

- Should we exclude Wood for these purposes?
Quote from Abdullah on Slack:
> Woods task complexity is different in the sense that it applies to task instances, not task classes.
For example, the room assignment task “class” can be classified using Steiner, McGrath, Shaw’s etc. The different room assignment task “instances” (specific number of students, rooms, constraints, and their relationships) vary in terms of their Wood’s complexity.
>The task space classifies task classes, not instances. Once we sample a task class to run the experiment, we will generate different instances of that task that vary in complexity.

- Should we all take a pass and do more wordsmithing / clean up which parts of the long elaborations from the text we want to use?
- Should we then put a version of these on Qualtrics (or perhaps --- for dogfooding purposes, just use a spreadsheet) and then dogfood by mapping a few tasks each? 
- If so, which tasks? (Pick at random, or identify some of the hardest tasks?)

**Comment by amaatouq** [comment]
> Should we exclude Wood for these purposes?

Yes, I think we should exclude Wood from the question. Wood's complexity applies to every point in the task space (i.e., task instances can vary in terms of Wood's complexity within the same task class). Maybe we want to crowdsource ideas from RAs on how to vary the complexity within a class, but it will not be used to figure out the location of a task class in the task space.

> Should we all take a pass and do more wordsmithing / clean up which parts of the long elaborations from the text we want to use?

I'll start doing some of this today and over the weekend.

> Should we then put a version of these on Qualtrics (or perhaps --- for dogfooding purposes, just use a spreadsheet) and then dogfood by mapping a few tasks each?

Maybe using the spreadsheet for now? Unless moving them to qualtrics is super easy.

> If so, which tasks? (Pick at random, or identify some of the hardest tasks?)
Maybe we can cherry-pick tasks (some of the hardest, but also some that are representative of the population of tasks that are extracted)?

**Comment by markwhiting** [comment]
Thanks @xehu — great! 

I'll also start taking a look sometime this week. I also agree re tasks to select, why not a few hard ones and one or two that we know well?

**Comment by xehu** [comment]
Awesome - thank you both so much!

**Comment by xehu** [comment]
So I made a preliminary list of tasks we could map --- it's a mix of the top tasks with contradictions + a handful of classic and familiar tasks.

**Comment by xehu** [comment]
And then I'm imagining that we can dogfood quickly in a spreadsheet like [this one](https://docs.google.com/spreadsheets/d/1MuEpgyCz3xnxg2Sx_9PyZv9ndOJKCXWBM9f22nnzDo8/edit#gid=0)? We'd just paste the new questions into the columns and rate the tasks in the other columns. To make sure we don't influence each other, we can do this separately with a different tab per question set and per person. In that case, we'd still see our own ratings for previous tasks that we mapped, but we wouldn't see each other's:

<img width="888" alt="Screen Shot 2021-12-16 at 5 46 39 PM" src="https://user-images.githubusercontent.com/28793641/146460181-c7a588a3-0b30-42f5-a8c0-49a95527d60b.png">

**Comment by markwhiting** [comment]
Awesome!
